internal/appsec/remoteconfig_test.go

```diff
@@ -371,3 +359,334 @@ func TestRemoteActivationScenarios(t *testing.T) {
 		require.False(t, Enabled())
 	})
 }
+
+func TestCapabilities(t *testing.T) {
+	for _, tc := range []struct {
+		name     string
+		env      map[string]string
+		expected []remoteconfig.Capability
+	}{
+		{
+			name:     "appsec-unspecified",
+			expected: []remoteconfig.Capability{remoteconfig.ASMActivation},
+		},
+		{
+			name: "appsec-enabled/default-rulesManager",
+			env:  map[string]string{enabledEnvVar: "1"},
+			expected: []remoteconfig.Capability{
+				remoteconfig.ASMRequestBlocking, remoteconfig.ASMUserBlocking, remoteconfig.ASMExclusions,
+				remoteconfig.ASMDDRules, remoteconfig.ASMIPBlocking,
+			},
+		},
+		{
+			name:     "appsec-enabled/rulesManager-from-env",
+			env:      map[string]string{enabledEnvVar: "1", rulesEnvVar: "testdata/blocking.json"},
+			expected: []remoteconfig.Capability{remoteconfig.ASMRequestBlocking, remoteconfig.ASMUserBlocking},
```

Comment:
I think this one is wrong. Let's talk with Nicolas tomorrow to make all of this clear.